NSS-Day-Cohort-53/kennels-siennas
# Added length to location.animals array

#### Changes Made

1. Modified file `LocationDetails.js` to include `animal count` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sc-AnimalandEmployeeCount
    ```
2. Open a new Terminal tab (⌘T or Ctrl+Shift+T) and navigate to the server directory.
3. Test app functionality.
    When user logs in and selects location tab, then user should see the current number animal at that location. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
